container/typings/LuigiContainer.svelte.d.ts

```diff
@@ -6,6 +6,13 @@ export declare interface UserSettings {
   [key: string]: number | string | boolean;
 }
 
+export declare interface WebComponentSettings {
+  id?: any;
+  type?: string;
+  selfRegistered?: boolean;
+  tagName?: string;
```

Comment:
I am not sure about the '?' making all the attributes optional. Are we sure all of these are optional. Maybe some must be required for the minimal example to work or its not the case ?

Reply:
I have checked it again. No property is required, each one is optional. Also an empty object works.